docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -221,7 +221,21 @@ Custom config is not yet supported for RocksDB.
 
 #### `--genesis` (string):
 
-Path to a JSON file containing the genesis data to use. Ignored when running standard networks (Mainnet, Fuji Testnet), or when `--genesis-content` is specified. If not given, uses default genesis data. For an example of a JSON representation of genesis data, see [here](https://github.com/ava-labs/avalanchego/blob/master/genesis/genesis_local.go#L16).
+Path to a JSON file containing the genesis data to use. Ignored when running standard networks (Mainnet, Fuji Testnet), or when `--genesis-content` is specified. If not given, uses default genesis data.
+
+This are the main properties in the JSON file:
+
+- `networkID`: A unique identifier for the blockchain, must be a number in the range [0, 2^32],
+- `allocations`: The list of initial addresses, their initial balances and the unlock schedule for each.
+- `startTime`: The time of the beginning of the blockchain, must be a timestamp in seconds,
+- `initialStakeDuration`: The initial stake duration in seconds,
+- `initialStakeDurationOffset`: The initial stake duration offset between stakers in seconds,
```

Comment:
Is it really offset between stakers?

Reply:
It's the offset between stakers regarding the end staking time: 
``` go
endTime := startTime + initialStakeDuration
node1.endStakingTime = endTime
node2.endStakingTime = endTime - 1 * offset
node2.endStakingTime = endTime - 2 * offset
node3.endStakingTime = endTime - 3 * offset
...
```

Reply:
Just curious about the purpose of this parameter, when is it useful?

Reply:
Risk of all stakers ending at the same time I guess